service-base/src/main/java/org/eclipse/hono/service/AbstractAdapterConfig.java

```diff
@@ -455,8 +455,8 @@ public HealthCheckServer healthCheckServer() {
      * @return The properties.
      */
     @Bean
-    @ConfigurationProperties(prefix = "hono.plan.prometheusBased")
-    @ConditionalOnProperty(name = "hono.plan.prometheusBased.host")
+    @ConfigurationProperties(prefix = "hono.plan.prometheus-based")
```

Comment:
these properties are defined in the documentation, which you'll need to adapt accordingly as well ...

Reply:
The above changes are needed only in the annotation side. The property `hono.plan.prometheus-based.host` or  `hono.plan.prometheusBased.host` can all be used as Spring Boot supports relaxed binding. So the properties such as  `hono.plan.prometheusBased.host` are still valid.

Reply:
That might be true. Still, our documentation should define the property name that we are actually using in the code, don't you think? It is an added bonus that people who (still) use a similar but different property also succeed ...

Reply:
yes, I will update it in the documentation too.

Reply:
I wasn't aware that this is required by Spring 2. I thought you deliberately chose to change the property prefix. Based on my new knowledge I think that we should keep the original camel case notation in our documentation for consistency. Would you mind changing back the property names in the docs to the original values? Sorry for the inconvenience ...

Reply:
I have removed the changes regarding property names in the docs.